sdkproject/Assets/Mapbox/Unity/Location/Location.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Mapbox.Unity.Location
+{
+	using Mapbox.Utils;
+	
+	/// <summary>
+	/// Location contains heading, latitude, longitude, accuracy and a timestamp.
+	/// </summary>
+	public struct Location
```

Comment:
@david-rhodes could you add a `Timestamp` property of type `DateTime` ?
I suppose `Mapbox.Unity.Location` is the object going to be passed over to probe extractor?

Reply:
In addition to the current timestamp? Would it be better just to create a static conversion method (unix <--> `DateTime`)

Reply:
@david-rhodes oops sorry, I totally missed the that there's already a `Timestamp` property 🙈 🙈 🙈 

Conversion methods are already here: https://github.com/mapbox/mapbox-unity-sdk/blob/7f9d203e38c9bbf205c6433bf9de08b8ed9b6a7b/sdkproject/Assets/Mapbox/Core/mapbox-sdk-cs/Utils/UnixTimestampUtils.cs